test/units.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+go get github.com/weaveworks/weave/testing/cover
```

Comment:
It seems very strange to refer to the weave gh repo here. Why can't this refer to the local version?